src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,18 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of validating:
```

Comment:
please move name of file out of "pre" to this line. (same for other example)

Reply:
Please check whether I have addressed this point correctly.